_posts/2019-10-10-server-0.6.0-released/index.md

```diff
@@ -0,0 +1,137 @@
+As a result, we gain a lot more efficient performance and resource utilization.
+Our benchmarks show that `t2.small` EC2 instance running a new version of Iglu Server can easily handle 1.000 concurrent requests per seconds without dropping of them.
+On the same instance, 0.5.0 version of Iglu Server starts to choke at 500 requests per second, either by dropping requests or responding within inappropriately long time. As a consequence, Snowplow pipeline slows down and sends some events into a bad bucket.
+
+Another big change on performance front is configurable thread and connection pools.
+While previous versions hardcoded default settings everywhere, the new Server gives a lot of configuration options to tune the performance.
+
+* Use single DB connection or [HikariCP][hikaricp] (recommended)
+* `connectionTimeout`, `maxLifetime`, `maximumPoolSize` settings from HikariCP
+* Configurable thread pool for DB transactions
+* Configurable thread pool for DB connections
+* Configurable thread pool for HTTP server
+
+All thread pools can have one of the following formats: `global` for global Scala [ForkJoinPool][fork-join-pool], `cached` for [cached JVM thread pool][cached-thread-pool] or `fixed N` for connection pool with fixed N amount of threads.
+
+Iglu Server provides sensible defaults for all these options, but its now possible to tune the performance for specific usage scenarios.
+
+<h3 id="api-changes">2. API changes</h3>
+
+We had two main aims for API changes in this release:
+
+1. Preserving core API for fetching schemas unchanged
+2. Reducing chance of accidental mistake by making API stricter
+3. Adding more flexibility
+
+There's not much to add for the first item - all our tools, including old versions of Snowplow pipeline, Snowplow Mini, igluctl (with very minor inconsistencies) and Iglu Clients are working with new version of Iglu Server just fine.
+If you're interested only in stability and performance improvements - you can proceed to [upgrade section](#upgrading).
+
+Previous versions of Iglu Server were extremelly permissive about inconsistent state, providing just basic CRUD operations without semantic checks.
+This could lead to varios hard to debug incidents with mutated or schemas and inconsistent warehouse state.
+In this release we tried to narrow down most common user mistakes and provide defensive checks for them in Iglu Server:
+
+* User cannot create a schema if previous one does not exist, e.g. if there's only `1-0-0` schema available and user tries to submit `1-0-2` it will result in 409 CONFLICT HTTP response
+* User cannot create a schema using HTTP PUT method if its URI and metadata do not correspond
+* User cannot create a private schema if previous ones were public and vice versa (otherwise there will be gaps between versions for public requests)
```

Comment:
```suggestion
* Users cannot create a private schema if previous ones were public and vice versa (otherwise there will be gaps between versions for public requests)
```